### ISSUE TITLE
Fatal error when running Tinker 2

### ISSUE THREAD

**Issue by samybasset** [bangnokia/tinker2#8]
Hi! 

I just purchased Tinker 2 yesterday. When I am running anycode (even returning primitives) the following error occurs: 

```php
Fatal error: Uncaught Error: Call to undefined function BangNokia\Psycho\tap() in phar:///Applications/Tinker 2.app/Contents/Resources/bin/psycho.phar/src/ClassAliasAutoloader.php:39
```

**Comment by bangnokia** [comment]
Hello Samy,

Can you give me more details about your current PHP version, and what kind of project you're using on (Laravel or Wordpress)?

**Comment by samybasset** [comment]
I am running php 8.0.13. And I want to run the project standalone, just using it as a repl basically

**Comment by bangnokia** [comment]
Thank you for your feedback, I found the problem when using with normal PHP project, `tap` function only working on laravel project atm. I'll try to give you new update as soon as possible. Thank you for your purchase even you can use it for free 😆

**Comment by bangnokia** [comment]
hi @samybasset ,
I just released new version 0.2.15 for Tinker 2 with bug fixing for binary `.phar` file. If error still happen and if your project is publicly, I can help take a look on it. You should reopen app to auto update.

**Comment by samybasset** [comment]
Sweet, the issue is fixed! Thanks for the quick fix 😄
